### PR TITLE
Flow examples

### PR DESCRIPTION
**Pull request by andreivladbrg**

### CODE REVIEW

**Comment by andreivladbrg** [comment]
smol-ninja surely, the wording can be polished, but I have marked this PR as ready for review

**Review by smol-ninja — APPROVED** [review]
Great work on the examples content, I enjoyed reading it. While reading through it, I made some changes related to formatting and wordings used. Please have a look, in case you find anything confusing or less appropriate, feel free to change it.

**Review by andreivladbrg — COMMENTED** [review]
thanks for the polishes, it looks good

File: docs/guides/flow/examples/03-create-stream.mdx [hunk 1]
```diff
@@ -23,7 +24,9 @@ Declare the Solidity version used to compile the contract:
 pragma solidity >=0.8.22;
 ```
 
-Import the relevant symbols from `@sablier/flow`, and the `FlowUtilities` library:
+Import the relevant symbols from
+[`@sablier/flow`](http://localhost:3000/guides/flow/examples/local-environment#install-flow-npm-package), and the
+[`FlowUtilities`](http://localhost:3000/guides/flow/examples/flow-calculate-rps) library:
```
[the remark this answers is not in the document]
Comment (smol-ninja) [reply docs/guides/flow/examples/03-create-stream.mdx]: Accidentally added them, removed now in https://github.com/sablier-labs/docs/pull/209/commits/00dc1fd18bcb73697644219f71b64326265541ef.

File: docs/guides/flow/examples/02-calculate-rps.mdx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+import { ud21x18, UD21x18 } from "@prb/math/src/UD21x18.sol";
+```
+
+Declare a library that can be used in other contracts:
+
+```solidity
+library FlowUtilities {}
+```
+
+## Calculate the Rate Per Second on a Duration
```
Comment (PaulRBerg) [inline docs/guides/flow/examples/02-calculate-rps.mdx:51]: Subtitles (`##` or below) are not in `Title Case`.

**Pull request merged**